internal/websocket/messageHandler.go

```
@@ -0,0 +1,218 @@
+		Stream string          `json:"stream"`
+		Data   json.RawMessage `json:"data"`
+	}
+
+	err := json.Unmarshal(msg, &combinedEvent)
+	if err == nil {
+		// Handle combined stream format
+		streamParts := strings.Split(combinedEvent.Stream, "@")
+		symbol := strings.ToUpper(streamParts[0])
+
+		var dataEvent struct{ E string }
+		json.Unmarshal(combinedEvent.Data, &dataEvent)
```

golangci-lint (1.62.2)

Error return value of `json.Unmarshal` is not checked

(errcheck)